app/src/main/java/com/example/spork/Restaurant.java

```diff
@@ -0,0 +1,136 @@
+package com.example.spork;
+
+import android.location.Location;
+
+import com.google.android.gms.maps.model.LatLng;
+import com.parse.ParseGeoPoint;
+import com.parse.ParseUser;
+
+import org.parceler.Parcel;
+
+@Parcel
+
+public class Restaurant {
```

Comment:
Noticing some overlap between this PR and #25. If you notice that there is a common aspect that two different branches use, it can be helpful to have one branch split off the other or have both branches split off the common aspect.

For example, you could have the restaurant be in it's own commit that both the prs depend on or have it in one of the prs and have the other branch off of it